docs: add fees section to integration docs (#551)
omisego/plasma-contracts: plasma_framework/docs/integration-docs/integration-doc.md

11. `txData` is unused and must be set to `0`
12. `metadata` must be padded to 32 bytes long

### Payment transaction fees

> Operator collects fees for including a transaction in a block. Transaction fee is implicit. It is the difference between transaction input utxos and output utxos, grouped by the token. For example if transaction inputs are the following utxos:

Reviewer: ```suggestion
Operator collects fees for including a transaction in a block. The transaction fee is implicit i.e. it is the difference between the transaction inputs and outputs, grouped by the token. For example if the transaction inputs are as follows:
```
